cmd/replay/main.go

```diff
@@ -54,7 +54,8 @@ func getBlockHeaderByHash(hash common.Hash) *types.Header {
 }
 
 type WavmChainContext struct {
-	chainConfig *params.ChainConfig
+	chainConfig   *params.ChainConfig
+	currentHeader *types.Header
 }
```

Comment:
the upstream commit pin has no reference to `header`, why are you adding this?

https://github.com/OffchainLabs/nitro/blob/v3.9.0/cmd/replay/main.go